code/src/test/pcgen/gui2/facade/CharacterLevelsFacadeImplTest.java

```diff
@@ -230,10 +231,7 @@ public void testFindNextLevelForSkillAllSameCost()
 			charLvlsFI.findNextLevelForSkill(umdSkill,
 				charLvlsFI.getElementAt(17), 1.0f));
 		// 6. Advise that the skill cannot be advanced.
-		assertEquals(
-			"Level for rank 21 spellcraft - cannot be advanced",
-			null,
-			charLvlsFI.findNextLevelForSkill(spellcraftSkill,
+		assertNull("Level for rank 21 spellcraft - cannot be advanced", charLvlsFI.findNextLevelForSkill(spellcraftSkill,
```

Comment:
what's the line limit? I wonder if this is a race been the line-length limit and the assertEquals -> assertNull fixer